pkg/notifiers/slack/handle_request.go

```diff
@@ -149,18 +149,21 @@ func (n *SlackNotifier) HandleRequestEvent(ctx context.Context, log *zap.Sugared
 				wg.Wait()
 			}
 		} else {
+
 			//Review not required
-			msg := fmt.Sprintf(":white_check_mark: Your request to access *%s* has been automatically approved.", requestedRule.Name)
+			msg := fmt.Sprintf(":white_check_mark: Your request to access *%s* in account has been automatically approved.\n", requestedRule.Name)
+
 			fallback := fmt.Sprintf("Your request to access %s has been automatically approved.", requestedRule.Name)
 			if err != nil {
 				return errors.Wrap(err, "building review URL")
 			}
-			n.SendDMWithLogOnError(ctx, log, request.RequestedBy, msg, fallback)
+
+			n.sendRequestDetailsMessage(ctx, log, request, requestedRule, *requestingUserQuery.Result, msg, fallback)
 		}
 	case gevent.RequestApprovedType:
-		msg := fmt.Sprintf("Your request to access *%s* has been approved.", requestedRule.Name)
+		msg := fmt.Sprintf(":white_check_mark: Your request to access *%s* has been approved.", requestedRule.Name)
```

Comment:
I think the same comment as above allies here on including rule name in the table instead